sdk/communication/Azure.Communication.Administration/src/Models/AdministrationModelFactory.cs

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Communication.Administration.Models
+{
+    /// <summary>
+    /// Model factory that enables mocking for the Administration library.
+    /// </summary>
+    public static class AdministrationModelFactory
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AcquiredPhoneNumber"/> class.
+        /// </summary>
+        /// <param name="phoneNumber"> String of the E.164 format of the phone number. </param>
+        /// <param name="acquiredCapabilities"> The set of all acquired capabilities of the phone number. </param>
+        /// <param name="availableCapabilities"> The set of all available capabilities that can be acquired for this phone number. </param>
+        /// <returns>A new <see cref="AcquiredPhoneNumber"/> instance for mocking.</returns>
+        public static AcquiredPhoneNumber AcquiredPhoneNumber(string phoneNumber, IEnumerable<PhoneNumberCapability> acquiredCapabilities, IEnumerable<PhoneNumberCapability> availableCapabilities) =>
+            new AcquiredPhoneNumber(phoneNumber, acquiredCapabilities, availableCapabilities);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AcquiredPhoneNumbers"/> class.
+        /// </summary>
+        /// <param name="phoneNumbers"> Represents a list of phone numbers. </param>
+        /// <param name="nextLink"> Represents the URL link to the next page. </param>
+        /// <returns>A new <see cref="AcquiredPhoneNumbers"/> instance for mocking.</returns>
+        public static AcquiredPhoneNumbers AcquiredPhoneNumbers(IReadOnlyList<AcquiredPhoneNumber> phoneNumbers, string nextLink) =>
+            new AcquiredPhoneNumbers(phoneNumbers, nextLink);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AreaCodes"/> class.
+        /// </summary>
+        /// <param name="primaryAreaCodes"> Represents the list of primary area codes. </param>
+        /// <param name="secondaryAreaCodes"> Represents the list of secondary area codes. </param>
+        /// <param name="nextLink"> Represents the URL link to the next page. </param>
+        /// <returns>A new <see cref="AreaCodes"/> instance for mocking.</returns>
+        public static AreaCodes AreaCodes(IReadOnlyList<string> primaryAreaCodes, IReadOnlyList<string> secondaryAreaCodes, string nextLink) =>
```

Comment:
nit: let's move all of the `=>` to the new line after the method signature

Reply:
Done, thanks!